### DISCUSSION TITLE
`Accessing non-existent property ...` warnings

### DISCUSSION THREAD

**Discussion by abrain** [renovatebot/renovate#28600] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub, Renovate 37.319.0

### Please tell us more about your question or problem

Since 37.317.0 the logs contain messages like `Accessing non-existent property ...`. I ran Renovate in debug mode (see below) and it happens right at the beginning when parsing the config.

It appears to be related to the addition of the `kbpgp` package in 37.317.0, as they have a bug report around that https://github.com/keybase/kbpgp/issues/204

It came up because now I get a cron email every hour with those four warning lines. They seem to escape the log redirection in this command:
`docker run --rm -v "/var/docker/renovate/config.js:/usr/src/app/config.js" -v "/var/docker/renovate/tmp:/tmp/renovate" renovate/renovate:37-full 2>&1 >> /var/log/renovate-bot.log`

### Logs (if relevant)

<details><summary>Logs</summary>

```

DEBUG: Using RE2 regex engine
DEBUG: Parsing configs
DEBUG: Checking for config file in config.js
(node:10) Warning: Accessing non-existent property 'b2u' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:10) Warning: Accessing non-existent property 'u2b' of module exports inside circular dependency
(node:10) Warning: Accessing non-existent property 'Pair' of module exports inside circular dependency
DEBUG: File config
       "config": {
         "autodiscover": false,
[...]

```

</details>

**Comment by viceice** [comment]
Working on it

**Reply by viceice** [reply, marked as answer]
- #28604

let's see if this is enough 🤞

**Reply by abrain** [reply]
Can confirm the warnings are gone. Thanks a lot :)
